src/lightly_train/_task_models/dinov2_ltdetr_object_detection/task_model.py

```diff
@@ -213,9 +207,13 @@ def predict(
         h, w = x.shape[-2:]
 
         x = transforms_functional.to_dtype(x, dtype=torch.float32)
+
```

Comment:
I believe you have to set `scale=True` in `to_dtype` to rescale to [0,1] before normalization